### PR TITLE
DESCW-601 Testing updates

### PR DESCRIPTION
**Pull request by codewisenate**
Blocks, patterns, and accessibility. Includes Skip-nav template part, media-text block filters & more.

### CODE REVIEW

File: inc/core/filters/media-text-enhanced.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		$defaults = [
+			'title' => 'none',
+		];
+
+		$args = wp_parse_args( $block['attrs'], $defaults );
+
+		if ( 'none' !== $args['title'] ) {
+			$title = 'title="' . esc_attr( $args['title'] ) . '"';
+		} else {
+			$title = null;
```
Comment (ShawnTurple) [inline inc/core/filters/media-text-enhanced.php:26]: just making sure this is your desired results as title = null. the logic afterwards seems a bit weird to at <img null

File: src/scripts/admin/filters/media-text-enhanced.js [hunk 2]
```diff
@@ -14,3 +22,71 @@ registerBlockStyle('core/media-text', {
 	name: 'no-clip',
 	label: 'Full Image',
 });
+
```
Comment (ShawnTurple) [inline src/scripts/admin/filters/media-text-enhanced.js:25]: seems like a lot of work to just add a title hover, at least WordPress is sticking to their core principles

File: src/scripts/public/sites/cleanbc/cleanbc-accessibility.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/**
+ * Accessibility DOM manipulation.
+ */
+const domReady = () => {
+	/*
+	 * SafarIE bug requires 0ms timeout.
```
Comment (ShawnTurple) [inline src/scripts/public/sites/cleanbc/cleanbc-accessibility.js:6]: maybe a link to the bug, might be helpful this sure seems annoying.

**Review by ShawnTurple — APPROVED** [review]

File: src/scripts/public/sites/cleanbc/cleanbc-accessibility.js (hunk 3 above)
Comment (codewisenate) [reply src/scripts/public/sites/cleanbc/cleanbc-accessibility.js:6]: It is an annoying race condition that tends to rear its head with Safari on iOS. Not sure if this is needed across the board, but it does delay DOM changes until the current call stack is emptied (which most browsers handle). As most of these scripts are handling DOM manipulation I skew towards using it so I don't later have to weed through the code to see where Safari falls down on the iPhone.

File: src/scripts/admin/filters/media-text-enhanced.js (hunk 2 above)
Comment (codewisenate) [reply src/scripts/admin/filters/media-text-enhanced.js:25]: Agreed.

File: inc/core/filters/media-text-enhanced.php (hunk 1 above)
Comment (codewisenate) [reply inc/core/filters/media-text-enhanced.php:26]: It is, but it could be structured differently for sure.

**Pull request merged**